app/helpers/application_helper/button/ovirt_ui.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationHelper::Button::OvirtUI < ApplicationHelper::Button::Basic
+  needs :@record
```

Comment:
Should we have here some logic to enable/disable or show/hide the button according to the type of virtual machine? It doesn't make much sense to have the button enabled or visible for VMware virtual machines, for example.

Reply:
Agreed, the button should be enabled only if the VM is managed by the oVirt provider.